src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.